docs/providers/aws/guide/serverless.yml.md

```diff
@@ -83,6 +83,7 @@ provider:
     binaryMediaTypes: # Optional binary media types the API might return
       - '*/*'
     metrics:  false # Optional detailed Cloud Watch Metrics
+    shouldStartNameWithService: true # Use `${service}-${stage}` naming for API Gateway. Will be `true` by default in next major version.
```

Comment:
In this doc, we need to expose defaults. So let's keep `false` (even though we explictly do not suport such setting). 
Technically we could set `null`, as that'll be valid on schema grounds, but then again it'll be confusing